boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "name": "DeepPrep",
+    "tool-version": "24.1.2",
+    "schema-version": "0.5",
+    "author": "personalized Brain Functional Seuctors (pBFS) Lab",
+    "description": "An accelerated, scalable, and robust pipeline for neuroimaging preprocessing empowered by deep learning",
+    "descriptor-url": "https://deepprep.readthedocs.io/en/latest/",
```

Comment:
not sure if it is a specific `tool-version`, should not it be https://deepprep.readthedocs.io/en/24.1.2/ ?